native_client/client.cc

```diff
@@ -87,10 +133,17 @@ main(int argc, char **argv)
   sox_close(input);
 
   // Pass audio to DeepSpeech
-  char* result = DsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
+  struct ds_result* result = LocalDsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
   if (result) {
-    printf("%s\n", result);
+    printf("%s\n", result->string);
+    if ((argc == 4) && (strncmp(argv[3], "-t", 2) == 0)) {
```

Comment:
It wouldn't matter too much, but should the length parameter in the strncmp be 3 so as to include the null byte? (otherwise it would match -tanything as well as -t?)